### PR TITLE
fix(timer): remove the code duplication in lv_timer_exec

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
### Description of the feature or fix

Minor code refactor

### Checkpoints
- [X] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [ ] Update [CHANGELOG.md](https://github.com/lvgl/lvgl/blob/master/docs/CHANGELOG.md)
- [ ] Update the documentation

### CODE REVIEW

File: src/misc/lv_timer.c [hunk 1]
```diff
@@ -315,15 +312,14 @@ static bool lv_timer_exec(lv_timer_t * timer)
...
+    }
 
-        /*Delete if it was a one shot lv_timer*/
-        if(timer_deleted == false) { /*The timer might be deleted by itself as well*/
-            if(timer->repeat_count == 0) {
-                TIMER_TRACE("deleting timer with %p callback because the repeat count is over", timer->timer_cb);
-                lv_timer_del(timer);
-            }
+    if(timer_deleted == false) { /*The timer might be deleted by itself as well*/
+        if(timer->repeat_count == 0) { /*Delete if it was a one shot lv_timer*/
```
Comment (kisvegabor) [inline src/misc/lv_timer.c]:
```suggestion
        if(timer->repeat_count == 0) { /*The repeat count is over, delete the timer*/
```

**Review by kisvegabor — COMMENTED** [review]
Thanks, only minor change in a comment.

File: src/misc/lv_timer.c (hunk 1 above)
Comment (xiaoxiang781216) [reply src/misc/lv_timer.c]: Done.

**Comment by kisvegabor** [comment]
Merging, thanks.

**Pull request merged**